tests/models/test_image_querying.py

```diff
@@ -581,8 +581,10 @@ def test_image_query(ptf_ref, decam_reference, decam_datastore, decam_default_ca
         assert len(results4) == 2
         assert results4[0].mjd == results4[1].mjd  # same time, as one is a coadd of the other images
         assert results4[0].instrument == 'PTF'
-        assert results4[0].type == 'ComSci'  # the first one out is the high quality coadd
-        assert results4[1].type == 'Sci'  # the second one is the regular image
+        # assert results4[0].type == 'ComSci'  # the first one out is the high quality coadd
+        # assert results4[1].type == 'Sci'  # the second one is the regular image
```

Comment:
This seems fine. This highlights the fact that if you are filtering by "quality" you will definitely get the coadds firsts, as they are almost always going to be much deeper that the regular images.

Reply:
and also you can remove the commented lines.